examples/bval-evaluation-redeployment/WebApp2/src/main/java/org/superbiz/webapp2/service/WebApp2Service.java

```diff
@@ -32,7 +31,7 @@
 public class WebApp2Service {
 
     @POST
-    public Response getInfo(@Pattern(regexp = "valid") final String input) {
```

Comment:
I believe which is needed, cause we are testing the validation with Apache BVAL.
Why we are removing it?

Reply:
The service consumes application/json but had a regexp for validation. Since the param key isn't defined, the pattern causes the request to generate a 406 not acceptable response. This is anticipated in the test, but it isn't proper, and there isn't any way to refine the test. Form/query/path/entity validation should be a separate example in its own right.

Reply:
The service consumes application/json but had a regexp for validation. Since the param key isn't defined, the pattern causes the request to generate a 406 response. This is anticipated in the test, and but there isn't any way to refine it. Form/query/path/entity validation should be a separate example in its own right.

Reply:
I'm sorry. I didn't realize BVal was a separate validation framework. That was a REALLY bad test before!

I understand it better now. Sorry for the confusion. I will revisit this, and see what components of BVal are applicable, and test them.

Reply:
After further review, I now see the bval-jsr dependency in openejb-core.

I think that was the cause of my confusion. It was implicitly available through openejb-core, but I didn't see it in the bval-evaluation-redeployment project.

It seems there is already some code coverage in openejb-core for testing bval. I'm uncertain whether or not we need an example for further testing here, but I will do as advised.

Reply:
yeah, just keep that test working with validation, then you can work in other samples.
Thank you so much @ShermanMarshall.

We really appreciate your contributions. No need apologies. :)

Reply:
I've fixed my changes back to the original. Thanks for the encouragement. 

I think I was confused because it looks like it accepts invalid JSON! I may have I made a change to the original before I understood how it worked, and that only caused more problems. I didn't understand why 406 was acceptable, changed the test, and then... it was all downhill. Good intentions, bad results :\

Reply:
Hmm... Maybe we found a bug? I don't know.
I'm not able to help exactly now, I've an appointment, but I'll try look at it during the weekend.

Thank you @ShermanMarshall for all your efforts.

Reply:
I'm not sure. I just didn't understand it at first. Speculation probably doesn't help, but I know invalid entities cause 406 errors. That is probably because of parsing the JSON, and this is just a String.

I think I have fixed this particular example. Setting the pom.xml build/finalName means the version number won't be relevant in the path. That solves potential future problems too! I hope that is acceptable. If you have further comments, I would be happy to hear them.